main.cpp

```diff
@@ -466,7 +467,17 @@ int main(int argc, char* argv[]) {
 
     // Test text
     glColor3f(0.1f, 0.2f, 0.5f);
-    text->draw(std::string("boats: ") + std::to_string(boats.size()) + std::string("\nsparks: ") + std::to_string(sparks.size()) + std::string("\nus/F: ") + std::to_string(uspf.count()));
+    
```

Comment:
I see ninja refactoring. :eyes:

If you're going to ninja @JayFoxRox, at least clean up your whitespace. ;P